app/integration/test_helpers.go

```diff
@@ -1,210 +0,0 @@
-package integration
-
-/**
-This file is full of test helper functions, taken from simapp
```

Comment:
I thought I imported this from some integration tests in `x/wasm`, but I guess we don't need it any more?

Reply:
ja, this was used with the IBC coordinator but can be added again when used.